Explorer/Assets/DCL/Roads/Settings/RoadSettingsAsset.cs

```diff
@@ -1,5 +1,8 @@
 using System;
 using System.Collections.Generic;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
```

Comment:
Why are you putting this code here and not in an editor assembly, DCL/Editor, for example?

Reply:
We discussed via DM, he is moving it to an Editor folder

Reply:
Juani already asked me to move it, consider it done. I usually (in my projects) put the custom editor code inside the affected class so no other class can access it, and everybody can easily realize that this class has a custom editor instead of having to look for it.